data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/source/KafkaSourceBufferAccumulator.java

```diff
@@ -8,7 +8,6 @@
 import com.fasterxml.jackson.core.JsonFactory;
```

Comment:
Do we need this? Taylor recently added generic buffer accumulator in the common directory. See `./data-prepper-plugins/buffer-common/src/main/java/org/opensearch/dataprepper/buffer/common/BufferAccumulator.java`

Reply:
This [/KafkaSourceBufferAccumulator.java] is specific to Kafka-source and it has a lot of utility methods to deal with the retry logic, offset commit update etc.